src/components/installedAppsView/InstalledAppsView.js

```diff
@@ -192,7 +192,7 @@ class InstalledAppsView extends React.Component {
             >
               <div className="integr8ly-state-ready">
                 <Button
-                  onClick={() => window.open(`${window.OPENSHIFT_CONFIG.masterUri}/console`, '_blank')}
+                  onClick={() => window.open(`${window.OPENSHIFT_CONFIG.masterUri}/dashboards`, '_blank')}
```

Comment:
Do we need a check here for openshift 3?

Reply:
Yes, good point. This still has to work on OS3. Will update.